taipy/gui/gui.py

```diff
@@ -1078,12 +1081,8 @@ def __send_var_list_update(  # noqa C901
                     newvalue = newvalue.get()
                 if isinstance(newvalue, (dict, _MapDict)):
```

Comment:
could combine the 2 if

Reply:
I thought of it too but want to keep the original structure. Will combine it anyway.